src/services/replyService.js

```diff
@@ -39,22 +43,23 @@ class ReplyService {
       params,
       parentId
     } = this.determineParamOptions(options);
-
+    console.log("the params are", params);
```

Comment:
remove console log. also, double quotes `""` are reserved for html. javascript should be single quotes `''`